sagemaker-inference-recommender/sklearn-inference-recommender/skleran-inference-recommender.ipynb

```diff
@@ -0,0 +1,715 @@
+{
```

Comment:
<p>Typo in filename - <code>skleran</code>  -&gt; <code>sklearn</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3065/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Fixed.